network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/dear_internet_letter_block.html

```diff
@@ -2,24 +2,29 @@
 {% load wagtailcore_tags wagtailimages_tags i18n %}
 
 {# Placeholder. Template to be updated in follow-up PRs. #}
-<div class="dear-internet-letter">
-  {% if value.image %}
-  {% image value.image original %}
-  {% endif %}
+<div class="dear-internet-letter-wrapper">
+  <div class="dotted-shadow"></div>
+  <div class="dear-internet-letter">
+    {% if value.image %}
+    {% image value.image original alt="" loading="lazy" class="mb-4 d-block mx-auto" %}
+    {% endif %}
 
-  {% if value.video_url %}
-  <p>video url: {{ value.video_url }}</p>
-  {% endif %}
+    {% if value.video_url %}
+    <p><small></small><a href="{{ value.video_url }}" target="_blank">{{value.video_url}}</a></small></p>
```

Comment:
```suggestion
    <p><small><a href="{{ value.video_url }}" target="_blank">{{value.video_url}}</a></small></p>
```
Although relying on `<small>` rather than a CSS class means browsers are free to do different things, so probably better to use a `small` class instead?

Reply:
fixed!